packages/react-search-ui-views/src/view-helpers/getUrlSanitizer.js

```diff
@@ -0,0 +1,33 @@
+const VALID_PROTOCOLS = ["http:", "https:"];
+
+/**
+ *
+ * @param {URL} URLParser URL interface provided by browser https://developer.mozilla.org/en-US/docs/Web/API/URL
+ * @param {String} currentLocation String representation of the browser's current location
+ */
+export default function getUrlSanitizer(URLParser, currentLocation) {
+  // This function is curried so that dependencies can be injected and don't need to be mocked in tests.
+  return url => {
+    if (!url) return url;
+
+    let parsedUrl;
+
+    try {
+      // Attempts to parse a URL as absolute
+      parsedUrl = new URLParser(url);
+      // eslint-disable-next-line no-empty
+    } catch (e) {}
```

Comment:
Just to make sure I'm on the same page, an empty catch swallows any error without failing/stopping correct? So we're basically just continuing if this fails?

Reply:
Yes